### DISCUSSION TITLE
Send websocket message to specific user

### DISCUSSION THREAD

**Discussion by rdzakmic** [strawberry-graphql/strawberry#3067] [Q&A]
I'm having scenario where i have plain subscription defined.
Through Info input parameter i can read user once subscription is made.
So if i have changes in database that are made and i wan't to notify specific user through subscription then how i can tell in Subscription method whom user should i send message?
Thanks for any suggesion

**Comment by DoctorJohn** [comment]
Hi. Are you using a framework to interface with your database? Some frameworks come with helpers which make this easier.

Generically, your subscription handler would await changes for the subscribing user and then send/yield them. In the most simple case "awaiting changes" can be done using a queue which your handler consumes from and other parts of your application feed in to.

However, more common solutions are based on something like Redis or framework specific components. You can take a look at our docs for Django Channels, which shows on (framework specific) approach.
